README.md

```diff
@@ -21,15 +21,15 @@ provider "ibm" {
 module "kms_root_key" {
   source  = "terraform-ibm-modules/kms-key/ibm"
   version = "latest" # Replace "latest" with a release version to lock into a specific release
-  kms_instance_id = ibm_resource_instance.kms_instance.guid
+  kms_instance_id = "XXxxXXxx-xxxx-XXXX-xxxx-XXxxXXxx"
   key_name                = "my-root-key"
 }
 
 # KMS standard key
 module "kms_standard_key" {
   source  = "terraform-ibm-modules/kms-key/ibm"
   version = "latest" # Replace "latest" with a release version to lock into a specific release
-  kms_instance_id = ibm_resource_instance.kms_instance.guid
+  kms_instance_id = "XXxxXXxx-xxxx-XXXX-xxxx-XXxxXXxx"
   key_name                = "my-standard-key"
   standard_key            = true
```

Comment:
Can we fix up the indentation so all the equal signs are matched up (like the linter would do).

Also replace "latest" with "X.X.X"

Reply:
in other TIM modules, we are using "latest" only - although in ibm_catalog.json its X.X.X